src/docs/development/ui/layout/index.md

```diff
@@ -1,1191 +1,52 @@
 
-* [Layout tutorial](/docs/development/ui/layout/tutorial)
-: Learn how to build a layout.
-* [Widget Overview](/docs/development/ui/widgets)
-: Describes many of the widgets available in Flutter.
-* [HTML/CSS Analogs in Flutter](/docs/get-started/flutter-for/web-devs)
-: For those familiar with web programming, this page maps HTML/CSS functionality
-  to Flutter features.
-* [Flutter Gallery][]
-: Demo app showcasing many Material Design widgets and other Flutter features.
-* [Flutter API documentation]({{api}})
-: Reference documentation for all of the Flutter libraries.
-* [Dealing with Box Constraints in Flutter](/docs/development/ui/layout/box-constraints)
-: Discusses how widgets are constrained by their render boxes.
-* [Adding Assets and Images in Flutter](/docs/development/ui/assets-and-images)
-: Explains how to add images and other assets to your app's package.
-* [Zero to One with Flutter]({{site.medium}}/@mravn/zero-to-one-with-flutter-43b13fd7b354)
-: One person's experience writing his first Flutter app.
+<iframe src="https://dartpad.dartlang.org/experimental/embed-new.html?id=b94c33c08f3031b5ef9181e215c2006a&fw=true" width="100%" height="400px"></iframe>
 
-[build()]: {{api}}/widgets/StatelessWidget/build.html
-[Card]: {{api}}/material/Card-class.html
-[Center]: {{api}}/widgets/Center-class.html
-[Column]: {{api}}/widgets/Column-class.html
-[Container]: {{api}}/widgets/Container-class.html
-[Elevation]: {{site.material}}/design/environment/elevation.html
-[Expanded]: {{api}}/widgets/Expanded-class.html
-[Flutter Gallery]: {{site.repo.flutter}}/tree/master/examples/flutter_gallery
-[GridView]: {{api}}/widgets/GridView-class.html
-[GridTile]: {{api}}/material/GridTile-class.html
-[Icon]: {{api}}/material/Icons-class.html
-[Image]: {{api}}/widgets/Image-class.html
-[layout widgets]: /docs/development/ui/widgets/layout
-[ListTile]: {{api}}/material/ListTile-class.html
-[ListView]: {{api}}/widgets/ListView-class.html
-[Material card]: {{site.material}}/design/components/cards.html
-[Material Design]: {{site.material}}/design
-[Material library]: {{api}}/material/material-library.html
-[Row]: {{api}}/widgets/Row-class.html
-[Scaffold]: {{api}}/material/Scaffold-class.html
-[SizedBox]: {{api}}/widgets/SizedBox-class.html
-[Stack]: {{api}}/widgets/Stack-class.html
-[Text]: {{api}}/widgets/Text-class.html
-[tutorial]: /docs/development/ui/layout/tutorial
-[widgets library]: {{api}}/widgets/widgets-library.html
-[Widget catalog]: /docs/development/ui/widgets
+In your code, you see that the middle child is not only wrapped in an Expanded widget, but also has a flex property of 2. `flex` is an interger that determines how much space a flex factor will occupy. You see in Demo3 that the middle image now appears twice as large as the other images. However, the image isn't twice as large; it's occupying twice as much space. Because the Expanded widget is a flex factor, you can determine how much space it'll occupy by adding a flex property. 
```

Comment:
"However, the image isn't twice as large, it's occupying twice as much space."
What's the difference? Why is this an important point?

Reply:
While I was writing this part, I could overhear someone telling me it's not about image size it's about space. So I tried (keyword 'tried') to incorporate something for both, but I'll go back and rework this part.